web/core/components/empty-state/empty-state.tsx

```
@@ -175,6 +183,14 @@ export const EmptyState: React.FC<EmptyStateProps> = observer((props) => {
           ) : (
             <h3 className="text-sm font-medium text-custom-text-400 whitespace-pre-line">{title}</h3>
           )}
+          {anyButton && (
+            <>
+              <div className="relative flex items-center justify-center gap-2 flex-shrink-0 w-full">
+                {renderPrimaryButton()}
+                {renderSecondaryButton()}
+              </div>
+            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)